### PR TITLE
feact(alert/callout): design update

### PR DESCRIPTION
**Pull request by krisantrobus**
Made changes to styles for both Alert and Callout based on the these designs:
- [Alert](https://www.figma.com/design/l2qtpQVMmXzQPVB3rEAA8i/Paste-Components-%26-Themes?node-id=119-2&node-type=CANVAS&t=cbK2Mo4u48OTfACk-0)
- [Callout](https://www.figma.com/design/l2qtpQVMmXzQPVB3rEAA8i/branch/S9gq6KMOQ7m0ckJWTMVc7N/Paste-Components-%26-Themes?node-id=14776-85818&node-type=FRAME&t=AZMZQ5atdpvXtXYQ-0)

The Callout component has additional functionality to dismiss the same way Alert currently does. Docs updated to include this example.

Design tokens were also updated to incorporate new design.

### CODE REVIEW

File: packages/paste-core/components/callout/package.json [hunk 1]
```diff
@@ -25,13 +25,17 @@
     "tsc": "tsc"
   },
   "peerDependencies": {
+    "@twilio-paste/anchor": "^12.0.0",
```
Comment (krisantrobus) [inline packages/paste-core/components/callout/package.json:28]: All of these are peer dependencies for including Button to Callout (for dismiss functionality).

File: packages/paste-core/components/callout/src/Callout.tsx [hunk 2]
```diff
@@ -43,33 +45,48 @@ export interface CalloutProps extends HTMLPasteProps<"div"> {
...
     backgroundColor: "colorBackgroundNewWeakest",
     color: "colorTextNew",
-    borderColor: "colorBorderNewWeaker",
+    borderColor: "colorBorderNewWeak",
   },
   neutral: {
     backgroundColor: "colorBackgroundNeutralWeakest",
     color: "colorTextNeutral",
-    borderColor: "colorBorderNeutralWeaker",
+    borderColor: "colorBorderNeutralWeak",
```
Comment (serifluous) [inline packages/paste-core/components/callout/src/Callout.tsx:89]: todo: All Callout backgrounds (except the error variant) should now be colorBackgroundWeak

File: packages/paste-core/components/callout/src/Callout.tsx [hunk 3]
```diff
@@ -115,6 +146,14 @@ export const Callout = React.forwardRef<HTMLDivElement, CalloutProps>(
         <Box display="flex" flexDirection="column" rowGap="space50" flex="1">
```
Comment (serifluous) [inline packages/paste-core/components/callout/src/Callout.tsx:146]: The variant icons should stack about the Callout text.

Comment (krisantrobus) [reply packages/paste-core/components/callout/src/Callout.tsx:142]: Oh man, no idea how I missed that. Sorry!! addressed in latest commits

File: packages/paste-core/components/callout/src/Callout.tsx [hunk 4]
```diff
@@ -101,20 +130,30 @@ export const Callout = React.forwardRef<HTMLDivElement, CalloutProps>(
...
+        borderLeftWidth="borderWidth20"
         variant={variant}
         {...variantStyles[variant]}
       >
-        <Box marginRight="space60" paddingTop="space10" element={`${element}_ICON`}>
-          {IconComponent}
-          <ScreenReaderOnly>{iconLabel}</ScreenReaderOnly>
-        </Box>
         <Box display="flex" flexDirection="column" rowGap="space50" flex="1">
+          <Box element={`${element}_ICON`}>
```
[the remark this answers is not in the document]
Comment (krisantrobus) [reply packages/paste-core/components/callout/src/Callout.tsx]: I think you've done a great job on the spec, I feel like I've just missed some pretty obvious things 🫤 sorry for that! I've pushed the changes up. There is a separation of 50 between icons and child elements and then a space of 30 between all child elements

File: packages/paste-core/components/alert/src/Alert.tsx [hunk 5]
```diff
@@ -22,12 +22,17 @@ export const AlertVariants = {
   NEUTRAL: "neutral",
   WARNING: "warning",
 } as const;
-export const AlertBackgroundColors = {
+export const AlertBackgroundColors: BoxProps["backgroundColor"] = {
```
Comment (nkrantz) [inline packages/paste-core/components/alert/src/Alert.tsx:25]: 👏

File: packages/paste-core/components/callout/src/Callout.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (krisantrobus) [reply packages/paste-core/components/callout/src/Callout.tsx:145]: Absolutely! Just pushed that change up

**Review by serifluous — APPROVED** [review]

**Pull request merged**